pages/linux/rmlint.md

```diff
@@ -0,0 +1,20 @@
+# rmlint
+
+> Extremely fast tool to remove remove duplicates and other lint from your filesystem.
+> More information: <https://rmlint.readthedocs.io/en/latest/index.html>.
+
+- Check directory for duplicates, empty files, and other lint:
```

Comment:
```suggestion
- Check a directory for duplicates, empty files, and other issues:
```

Reply:
Applied in [`b6375a1`](https://github.com/tldr-pages/tldr/pull/9090/commits/b6375a173de811384fa3df05af3ccadf79c1581e).